doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -173,6 +176,23 @@ concluded by the conjunction of their conclusions.
 
     Check tree_forest_mutind.
 
+.. example::
+
+  We can also combine schemes at sort ``Type``:
```

Comment:
Aligning the beginning of the paragraph with the word `example` would be the expected indentation here (similarly in the following with `coqtop`).